Engine/source/gui/controls/guiConsoleTextCtrl.cpp

```diff
@@ -115,6 +115,9 @@ void GuiConsoleTextCtrl::onPreRender()
    {
       mResult = Con::evaluatef( "$guiConsoleTextCtrlTemp = %s;", mConsoleExpression.c_str() );
```

Comment:
Please remove the `mResult =` here, it makes the control flow confusing to have two assignments to the same variable.